### ISSUE TITLE
Session Problem

### ISSUE THREAD

**Issue by manuelDistefano** [AndreasJonsson/magento-simplesamlauth#1]
Hello Andreas and thank you for share this module.

I'm able to connect to my IsP and authenticate bu i have a problem with session, it's like the user is never into the current session. I tried also to modify the controller and use some magento function like setCustomerAsLoggedIn() etc without any fortune.

I cant understand if i need to use sspmod_magentostore_Store_Mage, the class (construct or other methods) are never called. Do i need particular configuration for use it?

Thank you

**Comment by AndreasJonsson** [comment]
Yes, you will have to use the session plugin to have simplesamlphp share session with magento.  This is a plugin for simplesamlphp which you willl need to add to your simplesamlphp installation.  See the comment in the Mage.php file for instructions:

https://github.com/AndreasJonsson/magento-simplesamlauth/blob/master/simplesamlphp/modules/magentostore/lib/Store/Mage.php

**Comment by manuelDistefano** [comment]
Oh ok, now session work properly in some page (account, account edit information, etc) but not in others (cart for example). 

In config.php i have:
`    'magento.store.magentodir'      => '{{installation_path}}',
    'store.type'                    => 'magentostore:Mage',`

'magento.store.magentodir' exsist and is ok.

Plus i have 
`    'session.phpsession.cookiename' => 'custom_session_saml',
    'session.phpsession.savepath' => null,
    'session.phpsession.httponly' => true,`

Maybe i need some configuration here? Can you point me to the correct direction?^^

Tnx!!
